src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```diff
@@ -784,7 +784,7 @@ private void RegisterFunctionPointer(string functionName, MemberInfo method, Lis
             else if (CoreUtils.IsGetter(functionName))
             {
                 f = new GetterFunctionPointer(Module, functionName, method, retype);
-                (f as GetterFunctionPointer).ReflectionInfo.CheckForRankReductionAttribute(mGetterAttributes);
+                (f as GetterFunctionPointer).ReflectionInfo.CopyRankReductionAttribute(mGetterAttributes);
```

Comment:
Sorry, but I think the earlier name was more appropriate considering the context in which it is used.